solidity/test/isms/AmountRoutingIsm.t.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+
+import {TestIsm} from "../../contracts/test/TestIsm.sol";
+import {AmountRoutingIsm} from "../../contracts/isms/warp-route/AmountRoutingIsm.sol";
+import {TokenMessage} from "../../contracts/token/libs/TokenMessage.sol";
+import {Message} from "../../contracts/libs/Message.sol";
+import {IInterchainSecurityModule} from "../../contracts/interfaces/IInterchainSecurityModule.sol";
+
+contract AmountRoutingIsmTest is Test {
```

Comment:
Can we add a test that tests the ISM with a warp route?

Reply:
sure